app/views/request-to-go-live/organisation-name.njk

```diff
@@ -0,0 +1,28 @@
+{% extends "../layout.njk" %}
+{% from "components/input/macro.njk" import govukInput %}
+{% from "components/button/macro.njk" import govukButton %}
+{% from "components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle %}
+  Request to go live - {{ currentService.name }} - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-two-thirds govuk-!-margin-top-6">
+    <span id="request-to-go-live-current-step" class="govuk-caption-l">Step 1 of 3</span>
+    <h1 class="govuk-heading-l">What is the name of your organisation?</h1>
```

Comment:
Basically, move this into the Nunjucks for the input but like

```suggestion
    label: {
      text: "What is the name of your organisation?",
      isPageHeading: true,
      classes: "govuk-heading-l"
    }
```

Reply:
We tried that, but the result looks different
<img width="498" alt="screen shot 2019-01-16 at 17 25 27" src="https://user-images.githubusercontent.com/7883702/51266899-586f2f00-19b4-11e9-840d-318f3c235900.png">

Reply:
ah sorry try `classes: "govuk-label--xl"` which is how it is in the example here — https://design-system.service.gov.uk/patterns/question-pages/#label-as-a-page-heading

Reply:
It looks like we need to use two dashes for the class `govuk-label--l`, looks nice now, thanks!